.github/dependabot.yaml

```diff
@@ -0,0 +1,10 @@
+version: 2
```

Comment:
- Dependabot support must be enabled in order for this file to be useful
- We can add other entries in this file in order to update Go dependencies and Docker base images (i.e FROM)